src/app/modules/payments/stripe.utils.ts

```diff
@@ -43,6 +81,7 @@ export const getChargeIdFromNestedCharge = (
 const isStripeMetadata = (
   obj: Stripe.Metadata,
 ): obj is StripePaymentMetadataDto =>
+  // hasProp(obj, 'env') && // TODO: Make this required later
```

Comment:
can we create an issue tagged to the TODO so we don't forget about this later on?